spec/factories/proceeding_types.rb

```diff
@@ -49,6 +49,22 @@
       involvement_type_applicant { true }
     end
 
+    trait :as_prohibited_steps_order do
+      code { 'PH001' }
+      ccms_code { 'SE003' }
+      meaning { 'Prohibited Steps Order-S8' }
```

Comment:
:cry: Jim pointed out that I introduced typos on these seeds, it should be `Prohibited steps order`
 I'll pick up the amendments on mine when I rebase :+1: